sdk/core/azure-core/inc/azure.hpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+/**
+ * @brief Add main headers from Azure.
+ *
+ *
```

Comment:
```suggestion
 * @file
 * @brief Add main headers from Azure.
```
Without `@file`, the comment section may stick to the next random uncommented entity that happens to be there - doxygen does not know that you are talking about the header file here.